internal/batcher/batcher_test.go

```diff
@@ -112,31 +112,28 @@ func TestSaturation(t *testing.T) {
 func TestShutdown(t *testing.T) {
 	ctx := context.Background()
 	var nAdds int64 // atomic
-	b := New(int(0), 10, func(x interface{}) error {
+	c := make(chan int, 10)
+	b := New(int(0), cap(c), func(x interface{}) error {
```

Comment:
Is it possible to change the package to `batcher_test` so this would be `batcher.New`?

Reply:
Sorry, I only saw that this PR had been merged after I sent the comments.

Reply:
I did this in #791.